rust/lance-core/src/lib.rs

```diff
@@ -16,11 +18,20 @@ pub const ROW_ID: &str = "_rowid";
 /// Column name for the meta row address.
 pub const ROW_ADDR: &str = "_rowaddr";
 
+pub const LANCE_HOME_ENV_KEY: &str = "LANCE_HOME";
+pub const LANCE_HOME_DEFAULT_DIRECTORY: &str = "lance";
```

Comment:
Let's make this specific to lance + lindera

```suggestion
pub const LANCE_HOME_ENV_KEY: &str = "LANCE_LINDERA_HOME";
pub const LANCE_HOME_DEFAULT_DIRECTORY: &str = "lance_lindera";
```

Reply:
yes. global home path may be not a good idea. so I created a draft. But I think tokenizers can share same home.
different tokenizers are quite similar. they all contain
    1. trie dictionary, 
    2. ngram model, for chinese, 1-gram model is enough, so sometimes there's no this part.
 they all use viterbi algorithm.
that's why I think different tokenizers can share same home.

Reply:
I have moved env into lance-index package

Reply:
Okay. Maybe not `LANCE_LINDERA_HOME`. Maybe `LANCE_TOKENIZERS_HOME`?